server/src/Position.ts

```diff
@@ -0,0 +1,15 @@
+export default class Position {
+    x: number;
```

Comment:
If you fix the constructor, the member declarations can be removed

Reply:
Done.